src/app/features/users-page/users-page.component.html

```diff
@@ -1,5 +1,7 @@
 <a routerLink="/" class="back-link">Back To Home</a>
 <h1>List of all users:</h1>
+<button class="open-filter" (click)="openFilter()">Open Filter</button>
```

Comment:
always add `type` for buttons because buttons without type have `type=submit` which may unexpectedly send form

Reply:
not fixed